.travis.yml

```diff
@@ -2,6 +2,8 @@ language: php
 
 sudo: false
 
+dist: trusty
```

Comment:
please add this only for the jobs needing it (PHP 5.5)

Reply:
this would serve no purpose, the CI of Symfony 3.4 runs on trusty, all is fine:
https://github.com/symfony/symfony/blob/3.4/.travis.yml

Reply:
the purpose would be to use the newer Trusty environment in as many jobs as possible.